files/image_config/resolv-config/update-containers

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+for container in $(docker ps -a --format=" {{ .ID }}"); do
```

Comment:
Some containers like telemetry will start after 3 minutes.
If we upgrade SONiC image, and start the system for the first time, can we update /etc/resolv.conf in telemetry container?

Reply:
`docker ps -a` command returns all existing containers on the system. And we will update `/etc/resolv.conf` file for all containers including those that are not running.

For the case when the container is created in the runtime (for example app extension), when the dockerd starts the container it copies `/etc/resolv.conf` file from the host OS into the container filesystem. So, we will always have a valid DNS configuration inside the containers.